### PR TITLE
Add support for Network Address Types fixes: #50

### PR DESCRIPTION
**Pull request by andre1sk**

### CODE REVIEW

**Comment by bartlomieju** [comment]
Thanks! Can you add a test case testing those columns?

**Comment by andre1sk** [comment]
@bartlomieju  will do

**Comment by andre1sk** [comment]
@bartlomieju added tests

**Comment by andre1sk** [comment]
@bartlomieju are there any other changes needed?

File: tests/decode.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+test(function decodeInet() {
+  const column = new Column(
+    "inet_col", // name
+    16410,      // tableOid
+    1, // index
+    Oid.inet, // dataTypeOid
+    -1, // columnLength
+    -1, // typeModifier
+    Format.TEXT // format
+  );
```
Comment (bartlomieju) [inline tests/decode.ts]: I'm not really happy with that... you shouldn't create `Column` and fill it with info by hand. I'd much prefer integration test - create a table in test case that consist of these columns, then add a row and finally query the table to check if you get proper values back.

Comment (andre1sk) [reply tests/decode.ts]: @bartlomieju Sure do you want table to be created in constants.ts or ...? Should tests go into decode.ts or do you want something like data_types.ts that would have integration tests for data types?

Comment (bartlomieju) [reply tests/decode.ts]: `data_types.ts` sounds good :+1: just create table there for a single test, we can iterate on that later, thanks!

File: tests/data_types.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+const SETUP = [
+  "DROP TABLE IF EXISTS data_types;",
+  `CREATE TABLE data_types(
+     inet_t inet,
+     macaddr_t macaddr,
+     cidr_t cidr
+  );`
+];
+
```
Comment (bartlomieju) [inline tests/data_types.ts]: 👍 yep, this is great

**Comment by andre1sk** [comment]
@bartlomieju  should be ready to go

**Pull request merged**